packages/pigeon/lib/java_generator.dart

```diff
@@ -939,6 +950,14 @@ protected static ArrayList<Object> wrapError(@NonNull Throwable exception) {
     });
   }
 
+  void _writeCanIgnoreReturnValueAnnotation(
```

Comment:
I think it's worth a comment here explaining that we are emitting our own definition of `CanIgnoreReturnValue` to support clients who use `CheckReturnValue`, without having to force all Pigeon clients to take a new dependency on `error_prone_annotations`.